### PR TITLE
Zpls echogram generation

### PR DESCRIPTION
**Pull request by cwingard**
I worked through the generator and made several changes, mostly cosmetic for your review. The results can be seen via the URLs I put in the email from last week. Happy to talk through these changes this week.

### CODE REVIEW

File: zpls_echograms/zpls_echogram.py [hunk 1]
```diff
@@ -1,172 +1,374 @@
...
+        'colorbar_range': [-150, 0],
+        'vertical_range': [0, 450],
+        'deployed_depth': 450
+    },
+    'CP03ISSM': {
+        'long_name': 'Inshore Surface Mooring',
+        'tilt_correction': 15,
+        'colorbar_range': [-150, 0],
+        'vertical_range': [0, 90],
+        'deployed_depth': 90
```
Comment (snwhite) [inline zpls_echograms/zpls_echogram.py:59]: We should use the design depth of 95 as the default.

File: zpls_echograms/zpls_echogram.py [hunk 2]
```diff
@@ -1,172 +1,374 @@
...
+    },
+    'CP01CNSM': {
+        'long_name': 'Central Surface Mooring',
+        'tilt_correction': 15,
+        'colorbar_range': [-150, 0],
+        'vertical_range': [0, 130],
+        'deployed_depth': 130
```
Comment (snwhite) [inline zpls_echograms/zpls_echogram.py:66]: We should use the design depth of 135 as the default.

File: zpls_echograms/zpls_echogram.py [hunk 3]
```diff
@@ -1,172 +1,374 @@
...
+        'tilt_correction': 15,
+        'colorbar_range': [-150, 0],
+        'vertical_range': [0, 150],
+        'deployed_depth': 150
+    },
+    'GI02HYPM_Lower': {
+        'long_name': 'Apex Profiler Mooring, Downward Looking',
+        'tilt_correction': 15,
+        'colorbar_range': [-150, 0],
+        'vertical_range': [150, 300],
```
Comment (snwhite) [inline zpls_echograms/zpls_echogram.py:79]: Colorbar range for Globals should be tightened up.  CG will need to determine what works.
If this is now vertical range, the downward-looking units should be 0 to something.... 500?

File: zpls_echograms/zpls_echogram.py [hunk 4]
```diff
@@ -1,172 +1,374 @@
...
+        'tilt_correction': 15,
+        'colorbar_range': [-150, 0],
+        'vertical_range': [0, 150],
+        'deployed_depth': 150},
+    'GP02HYPM_Lower': {
+        'long_name': 'Apex Profiler Mooring, Downward Looking',
+        'tilt_correction': 15,
+        'colorbar_range': [-150, 0],
+        'vertical_range': [150, 300],
+        'deployed_depth': 150
```
Comment (snwhite) [inline zpls_echograms/zpls_echogram.py:93]: We should include the GA and GS HYPM moorings.

**Review by snwhite — COMMENTED** [review]
I added a few comments for CG to address in our next update.

**Review by asmith75218 — APPROVED** [review]

**Pull request merged**